TOC.md

```diff
@@ -181,51 +181,22 @@
     - [TiKV Control](https://github.com/tikv/tikv/blob/master/docs/tools/tikv-control.md)
     - [TiDB Controller](tools/tidb-controller.md)
     - [工具下载](tools/download.md)
-  - [TiDB 路线图](ROADMAP.md)
-  + 用户案例
-    - [北京银行](http://t.cn/RnY8fGn)
-    - [海航](http://t.cn/REXx0Qe)
-    - [美团点评](http://t.cn/EAFCqhl)
-    - [今日头条](http://t.cn/RnLfEMf)
-    - [转转](http://t.cn/R1MAXEq)
-    - [Mobike](http://t.cn/RT8FbP6)
-    - [小米科技](http://t.cn/Ey2xCDK)
-    - [爱奇艺](http://t.cn/EvErsc1)
-    - [易果生鲜](http://t.cn/RTYVhzH)
-    - [同程旅游（一）](http://t.cn/RmXeNKR)
-    - [同程旅游（二）](http://t.cn/EAmsF08)
-    - [去哪儿](http://t.cn/RTKnsL7)
-    - [火星文化](http://t.cn/EAuvfcs)
-    - [G7](http://t.cn/RQVePoX)
-    - [一面数据](http://t.cn/RT9r5di)
-    - [凤凰网](http://t.cn/RHRQfNT)
-    - [猿辅导](http://t.cn/RTKnKSX)
-    - [Mobikok](http://t.cn/Rm1F6lg)
-    - [二维火](http://t.cn/R8bXM2f)
-    - [客如云](http://t.cn/R1wSEJH)
-    - [Ping++](http://t.cn/RE5xYKn)
-    - [乐视云](http://t.cn/Rnv3IVs)
-    - [零氪科技](http://t.cn/REj7tSv)
-    - [威锐达测控](http://t.cn/R3CrviR)
-    - [盖娅互娱](http://t.cn/RT9r7hx)
-    - [游族网络](http://t.cn/R8k4AWB)
-    - [西山居](http://t.cn/RBP12zj)
-    - [FUNYOURS JAPAN](http://t.cn/Rnoab5D)
-    - [丰巢科技](http://t.cn/EAuvLIv)
-    - [特来电](http://t.cn/RrHzUGW)
-    - [万达网络](http://t.cn/RTKm6ds)
-    - [360金融](http://t.cn/RTKnTev)
-    - [中国电信翼支付](http://t.cn/R3Wd9p3)
-    - [某电信运营商](http://t.cn/RTYWADg)
 + 常见问题 (FAQ)
   - [TiDB FAQ](FAQ.md)
   - [TiDB-Lightning FAQ](tools/lightning/faq.md)
   - [升级 FAQ](op-guide/upgrade-faq.md)
-+ [版本发布历史](releases/rn.md)
-  + v3.0
+- 技术支持
+  - [支持渠道](support.md)
+  - [反馈问题](report-issue.md)
+- [贡献](contribute.md)
+  - [贡献代码](contribute.md#成为-tidb-的贡献者)
+  - [改进文档](contribute.md#改进文档)
+- [TiDB 路线图](ROADMAP.md)
+- [版本发布历史](releases/rn.md)
+  - v3.0
     - [3.0.0 Beta.1](releases/3.0.0-beta.1.md)
     - [3.0 Beta](releases/3.0beta.md)
-  + v2.1
+  - v2.1
```

Comment:
The original `+` is on purpose, to make it easier to read this TOC file. `+` indicates it has sub-levels. I'll change it back.